### PR TITLE
Fix issue #116

### PR DESCRIPTION
**Pull request by mohammedzee1000**
This is a fix for the issue https://github.com/CentOS/CentOS-Dockerfiles/issues/116

### CODE REVIEW

**Comment by mohammedzee1000** [comment]
ping @jperrin

**Comment by mohammedzee1000** [comment]
@surajssd fyi

File: rabbitmq/centos7/Dockerfile [hunk 1]
```diff
@@ -43,7 +47,7 @@ ADD passwd.template /tmp/rabbitmq/passwd.template
 RUN chown -R rabbitmq:0 /tmp/rabbitmq && chmod -R ug+rwx /tmp/rabbitmq
```
Comment (scara) [inline rabbitmq/centos7/Dockerfile]: Should change `rabbitmq` into `1001`.

Comment (mohammedzee1000) [inline rabbitmq/centos7/Dockerfile]: Thanks will update, hopefully wont affect function.

Comment (scara) [inline rabbitmq/centos7/Dockerfile]: Yep 😉

File: rabbitmq/centos7/Dockerfile [hunk 2]
```diff
@@ -7,13 +7,30 @@
...
-#
-# add run/set passwd script
-ADD run-rabbitmq-server.sh /run-rabbitmq-server.sh
-RUN chmod 750 ./run-rabbitmq-server.sh
+ADD run-rabbitmq-server.sh /tmp/rabbitmq/run-rabbitmq-server.sh
+
+# Set permissions for openshift run
+RUN chown -R 1001:0 /etc/rabbitmq && chown -R 1001:0 /var/lib/rabbitmq  && chmod -R ug+rw /etc/rabbitmq && chmod -R ug+rw /var/lib/rabbitmq
+RUN find /etc/rabbitmq -type d -exec chmod g+x {} +
+RUN find /var/lib/rabbitmq -type d -exec chmod g+x {} +
```
Comment (surajssd) [inline rabbitmq/centos7/Dockerfile]: can we reduce all those layers? I think most of these RUN statements can be clubbed into one or max two RUN statements?

Comment (mohammedzee1000) [inline rabbitmq/centos7/Dockerfile]: Will do updating

**Comment by mohammedzee1000** [comment]
@surajssd updated

**Comment by mohammedzee1000** [comment]
ping @jperrin @kbsingh i believe this is ready

**Comment by containscafeine** [comment]
@mohammedzee1000 tried this out, works for me, LGTM!

**Comment by mohammedzee1000** [comment]
@containscafeine @surajssd fyi the issue itelf has logs at the end showing the container is functional.

**Comment by containscafeine** [comment]
@mohammedzee1000 nit: maybe squash the commits :)

**Comment by mohammedzee1000** [comment]
Well there are no useless commits and ideally you dont want to squash unless there are intermediate commits directly related to something above them.  Should i still squash?

**Comment by containscafeine** [comment]
@mohammedzee1000 was just pointing out to keep the commit history better, but works either way. Just a nit.

**Comment by scara** [comment]
@mohammedzee1000 Usually commits should be atomic compared to the functionality they are supposed to fix/improve: here, the issue is that the container cannot work within Openshift and any of your commits properly addresses it showing your progresses w/ your own work and the advices/thoughts of the reviewers.
If I could vote, my +1 to squash them into 1 single commit reviewing the comments to be included (kind of intelligent `fixup`).

BTW Keep up the good work 👍 😄.

HTH,
Matteo

**Comment by surajssd** [comment]
@mohammedzee1000 yes LGTM :)

**Comment by mohammedzee1000** [comment]
Ok squashing

**Comment by mohammedzee1000** [comment]
Commits squashed @kbsingh good to merge

File: rabbitmq/centos7/Dockerfile [hunk 3]
```diff
@@ -7,13 +7,32 @@
...
+RUN yum -y install epel-release && yum -y update && yum -y install pwgen wget logrotate && yum -y install nss_wrapper gettext && yum clean all
+
+# Setup rabbitmq-server
+RUN mkdir -p /var/lib/rabbitmq && useradd -d /var/lib/rabbitmq -u 1001 -o -g 0 rabbitmq && \
+    yum -y install rabbitmq-server && yum clean all
+
+# Send the logs to stdout
+ENV RABBITMQ_LOGS=- RABBITMQ_SASL_LOGS=-
+
+VOLUME /var/lib/rabbitmq
```
Comment (containscafeine) [inline rabbitmq/centos7/Dockerfile]: @mohammedzee1000 please refer to https://github.com/CentOS/CentOS-Dockerfiles/issues/116#issuecomment-274228862, this does not fix the issue for me :(

**Pull request merged**